library/dnet.c

```diff
@@ -314,6 +314,31 @@ int dnet_send_reply(void *state, struct dnet_cmd *cmd, const void *odata, unsign
 	return err;
 }
 
+void dnet_queue_wait_threshold(void *state)
+{
+	struct dnet_net_state *st = state;
```

Comment:
I used this function in implementing server-send command in eblob originally. It is not needed now and can be made static